cfme/tests/containers/test_basic_metrics.py

```diff
@@ -117,3 +117,15 @@ def test_flash_msg_not_contains_html_tags(provider):
         is_translated_to_html = True
 
     assert is_translated_to_html, "Flash massage contains HTML tags"
+
+
+def test_typo_in_metrics_endpoint_type(provider):
+    """
+    This test based on bz1436698
```

Comment:
**Question**: Is this the correct bug?  It looks like this bug has to do with a report header being wrong